ml/models/logo_detection/detect.py

```diff
@@ -0,0 +1,144 @@
+import os
+import time
+from pathlib import Path
+import torch
+
+from yolov7_package import Yolov7Detector
+from yolov7_package.models.experimental import attempt_load
+from yolov7_package.utils.general import check_img_size, check_requirements, check_imshow, non_max_suppression, apply_classifier, \
+    scale_coords, xyxy2xywh, strip_optimizer, set_logging, increment_path
+from yolov7_package.utils.torch_utils import select_device, load_classifier, time_synchronized, TracedModel
+from yolov7_package.utils.datasets import LoadStreams, LoadImages
+
+from data import BoundingBox, LogoDetectionInference
```

Comment:
To integrate with the rest of the app we will need to use relative imports here to avoid import errors from other files
```suggestion
from .data import BoundingBox, LogoDetectionInference
```